boards/nxp/imx943_evk/Kconfig.defconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright 2025 NXP
```

Comment:
I think from a bisect-ability perspective, this commit should move to after the commits that updates the modules folder.

Reply:
changed the patch sequence, thanks.